Doc/library/asyncio-task.rst

```diff
@@ -109,34 +109,53 @@ To actually run a coroutine asyncio provides three main mechanisms:
 Awaitables
 ==========
 
-We say that an object is an *awaitable* object if it can be used
-in an :keyword:`await` expression.
+We say that an object is an **awaitable** object if it can be used
+in an :keyword:`await` expression.  Many asyncio APIs are designed to
+accept awaitables.
 
+There are three main types of *awaitable* objects:
+**coroutines**, **Tasks**, and **Futures**.
 
-.. rubric:: Coroutines and Tasks
 
-Python coroutines are *awaitables*::
+.. rubric:: Coroutines
+
+Python coroutines are *awaitables* and therefore can be awaited from
+other coroutines::
 
     async def nested():
         return 42
 
     async def main():
-        # Will print "42":
-        print(await nested())
+        # Nothing happens if we just call "nested()":
+        nested()  # a coroutine object is created but it is
+                  # not awaited and not wrapped into a Task.
```

Comment:
small nit - we talk about wrapping coroutines in the next task section

Also, it would be helpful and likely less confusing for those learning asyncio to have the import statements and `asyncio.run(main())` included in the examples so the user can run easily

Reply:
Fixed!